CMakeLists.txt

```diff
@@ -91,6 +91,7 @@ endif()
 
 include_directories(BEFORE ${CMAKE_CURRENT_BINARY_DIR}/include/)
 include_directories(BEFORE ${CMAKE_CURRENT_SOURCE_DIR}/include/)
+include_directories(SYSTEM ${CMAKE_CURRENT_SOURCE_DIR}/third_party/)
```

Comment:
Already rebased, which is how I picked up on the missing sol directory includes. I'll update how the sol header is saved and update in the cmakelists now.